config/examples/dot_product.yaml

```
@@ -0,0 +1,62 @@
+---
+type: config
+log_level: DEBUG
+cpu_affinity: [0,1,2,3]
+num_local_freq: 1
+samples_per_data_set: 4608
```

Strange length?  Let's make it simple.
```suggestion
samples_per_data_set: 1024
```